### PR TITLE
Add Resources for CRUD Endpoints Roles & Sites

### PR DESCRIPTION
**Pull request by qoda**
Add Resources for CRUD Endpoints Roles & Sites

### CODE REVIEW

File: admin/src/defaults.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const cardActionStyle = {
+    zIndex: 2,
+    display: 'inline-block',
+    float: 'right'
+};
+
+export const ListActions = ({ resource, filters, displayedFilters, filterValues, basePath, showFilter }) => (
+    <CardActions style={cardActionStyle}>
+        {filters && React.cloneElement(filters, { resource, showFilter, displayedFilters, filterValues, context: 'button' }) }
```
Comment (jwfwessels) [inline admin/src/defaults.js]: not sure why you are using a react.clone here? please elaborate 😄

File: admin/src/domains.js [hunk 2]
```diff
@@ -1,40 +1,19 @@
...
-    </CardActions>
-);
 
 const DomainTitle = ({ record }) => {
     return <span>Domain {record ? `'${record.name}'` : ''}</span>;
 };
 
 export const DomainList = (props) => (
-    <List actions={<DomainListActions />} title='Domains' {...props}>
+    <List actions={<ListActions />} title='Domains' {...props}>
```
Comment (jwfwessels) [inline admin/src/domains.js]: so does the actions property expect a Component? you should be able to just write
```actions={ListActions}```

File: admin/src/defaults.js (hunk 1 above)
Comment (qoda) [reply admin/src/defaults.js]: Nor do I, this is how it's done in the docs: https://marmelab.com/admin-on-rest/List.html

File: admin/src/domains.js (hunk 2 above)
Comment (qoda) [reply admin/src/domains.js]: It expects a component.

**Pull request merged**